.env.example

```diff
@@ -2,4 +2,5 @@ REACT_APP_LUMINUS_CLIENT_ID=your_luminus_client_id_here
 REACT_APP_VERSION=$npm_package_version
 REACT_APP_BACKEND_URL=http://localhost:4001
 REACT_APP_USE_BACKEND=TRUE
-REACT_APP_CHATKIT_INSTANCE_LOCATOR=instance_locator_here_otherwise_empty_string
\ No newline at end of file
+REACT_APP_CHATKIT_INSTANCE_LOCATOR=instance_locator_here_otherwise_empty_string
+REACT_APP_GOOGLE_CLIENT_ID=352013967309-qmhsu2ddofc79hpa0r3fe4q2djl16d7m.apps.googleusercontent.com
```

Comment:
could we remove the actual `client_id`. the example file should not contain any legitimate credentials!

Reply:
Fixed.